src/main/java/net/snowflake/client/jdbc/SnowflakeDriver.java

```diff
@@ -54,6 +57,10 @@ public class SnowflakeDriver implements Driver {
      * Get the manifest properties here.
      */
     initializeClientVersionFromManifest();
+
+    securityUtil = new SecurityUtil();
```

Comment:
can we get rid of the field and just call `SecurityUtil.addBouncyCastleProvider()`?